### DISCUSSION TITLE
Overflowing top nav bar long title

### DISCUSSION THREAD

**Discussion by joppehoekstra** [requarks/wiki#3648] [Error / Bug Report]
### Actual behavior
The navigation bar overflows when the title of the site is longer.

### Expected behavior
Not all sites have a short name such as ‘WikiJS’, so the site should be able to handle longer titles. One could, for example, hide the title on mobile devices, or break off the text by using the CSS property ‘text-overflow: ellipsis’ for the title.

### Steps to reproduce the behavior
In the admin control, change the title of the site to a longer name. On a mobile device, the top navigation bar will then overflow and certain buttons on the right will become inaccessible. Depending on how long the title is, this happens always or just when editing a page (because the edit page has more buttons).

As a word of thanks to developers: keep up the good work!

**Comment by Byrd910** [comment]
I'd like to add my support to this - my site title gets truncated on mobile as well!
